docs/src/main/paradox/release-notes/1.0-RC1.md

```diff
@@ -0,0 +1,69 @@
+# Alpakka 1.0-RC1
+
+## First release candidate of Alpakka 1.0
+
+By now all modules have been worked with to follow a similar structure so that both the user experience and the contributor experience are transferable across the many modules in Alpakka. The recommended structure for connectors is illustrated by the [Reference connector](https://doc.akka.io/docs/alpakka/current/reference.html).
+
+With this release candidate Alpakka reaches the end of the [Road to Alpakka 1.0](https://akka.io/blog/news/2018/08/30/alpakka-towards-1.0) we from now now we'll keep API changes to major releases, and keep minor releases [binary-compatible](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#binary-compatibility-rules).
+
+Having that said, a few parts make use of the `@ApiMayChange` annotation to keep the door open for API changes, so that they can evolve more rapidly than other modules within Alpakka.
+
+If you have kept to older versions of Alpakka to avoid multiple upgrades, **now is the time** to start migrating to what it offers with 1.0.
+
+Most parts of Alpakka are available for Scala 2.11, 2.12 and 2.13.0-M5.
+
+
+## Highlights in this release
```

Comment:
```suggestion
## Highlights since 1.0-M3
```

Reply:
This is helpful for users, that have not seen milestone releases before. So it becomes clear that for more changes release notes of 1.0-M3 should be considered.